### PR TITLE
Updated xexun sos alarm

### PR DESCRIPTION
**Pull request by redowk**
Added "help me" option, for sos alarm alert to the xexun protocol.
Following the discussion and tests described here: https://www.traccar.org/forums/topic/xexun-implement-sos-alarms

### CODE REVIEW

File: src/main/java/org/traccar/protocol/XexunProtocolDecoder.java [hunk 1]
```diff
@@ -79,6 +79,9 @@ private String decodeStatus(Position position, String value) {
                 case "help me!":
                     position.set(Position.KEY_ALARM, Position.ALARM_SOS);
                     break;
+                case "help me":
+                    position.set(Position.KEY_ALARM, Position.ALARM_SOS);
```
Comment (tananaev) [inline src/main/java/org/traccar/protocol/XexunProtocolDecoder.java]: You should not duplicate code here. Add second switch case for exiting SOS alarm option.

Comment (redowk) [reply src/main/java/org/traccar/protocol/XexunProtocolDecoder.java]: You mean a new switch case with only one option? Why?
```            switch (value.toLowerCase()) {
                case "help me":
                    position.set(Position.KEY_ALARM, Position.ALARM_SOS);
                    break;
                default:
                    break;
            }```

Comment (tananaev) [reply src/main/java/org/traccar/protocol/XexunProtocolDecoder.java]: This is what I meant:
```java
case "help me!":
case "help me":
    position.set(Position.KEY_ALARM, Position.ALARM_SOS);
    break;
```

Comment (redowk) [reply src/main/java/org/traccar/protocol/XexunProtocolDecoder.java]: sorry! I understand.
Thanks for your patience!

**Pull request closed without merging**